### PR TITLE
Added named parameter support

### PR DESCRIPTION
**Pull request by h-chaturvedi**

### CODE REVIEW

**Comment by smtakeda** [comment]
Doesn't this require libsnowflakeclient change as well? or already implemented before?

**Comment by h-chaturvedi** [comment]
> Doesn't this require libsnowflakeclient change as well? or already implemented before?

libsnowflakeclient changes are already implemented and committed in [this commit](https://github.com/snowflakedb/libsnowflakeclient/commit/2294764ff2fec6635fb2c290fed50bd3eb1d7bb8).

**Comment by smtakeda** [comment]
oh, sounds good. I saw the code somewhere but was not sure.

File: snowflake_paramstore.c [hunk 1]
```diff
@@ -0,0 +1,104 @@
+/*
```
Comment (notkriswagner) [inline snowflake_paramstore.c]: I assume this is just copied over from libsnowflakeclient

Comment (h-chaturvedi) [reply snowflake_paramstore.c]: Mostly yes. The Data Structures are copied. Slight changes to some functions and return types. Most new changes are in snowflake_stmt.c

**Pull request merged**